pkg/controller/instance/instance_controller.go

```diff
@@ -168,7 +166,7 @@ func isForPipePod(e event.DeleteEvent) bool {
 func (r *Reconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 	// ---------- 1. Query the current state ----------
 
-	log.Printf("InstanceController: Received Reconcile request for instance \"%+v\"", request.Name)
+	log.Printf("InstanceController: Received Reconcile request for instance %s", request.Name)
```

Comment:
Consistency nitpick: The `log.Printf` below is using `request.NamespacedName`. Let's agree on a consistent choice.

Reply:
I'm 💯 on board with consistency!